static/app/test/specs/entities.spec.js

```diff
@@ -8,27 +8,102 @@ var Radio = require('backbone.radio');
 
 // App
 var EntitiesAPI = require('app/entities').API;
+var Move = require('app/entities').Move;
 var channel = Radio.channel('global');
 
 describe('Entities', function(){
     describe('API', function(){
         describe('initialization', function(){
-            it('registers an entities:spots request handler');
-            it('registers an entities:moves request handler');
-            it('registers an entities:move request handler');
-            it('registers an entities:move:reset request handler');
+            
+            it('registers an entities:spots request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                var spots = "something";
+                this.replySpotsStub = sinon.stub(EntitiesAPI.prototype, 'replySpots', function(){ return spots });
```

Comment:
Looks like I was stubbing out the method on the prototype on the example I gave in constants.spec.js. Sorry for doing that!